src/App.tsx

```diff
@@ -1,24 +1,193 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
+
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import * as todosService from './api/todos';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoItem } from './components/TodoItem/TodoItem';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 11604;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [status, setStatus] = useState<Status>(Status.All);
+  const [title, setTitle] = useState('');
+  const [statusResponse, setStatusResponse] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)